packages/auto-label/src/auto-label.ts

```diff
@@ -17,6 +17,19 @@ import {Storage} from '@google-cloud/storage';
 import {Application, Context} from 'probot';
 import {logger} from 'gcf-utils';
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const langlabler = require('./language');
+// Default app configs if user didn't specify a .config
+const LABEL_PRODUCT_BY_DEFAULT = true;
+const LABEL_LANGUAGE_BY_DEFAULT = false;
+const DEFAULT_CONFIGS = {
```

Comment:
This may just be something to think about, perhaps no action required. Sometimes we install bots on an org-level (like autolabel), and the config file at that point dictates the configuration for all the repos. It seems like this configuration is assuming a repo-by-repo level, but perhaps you'd want to support a configuration on an org-level as well?

Reply:
I doubled checked: by default, probot imports configs from the org if it's not available at the repo level: 
`context.config supports sharing configs between repositories. If configuration for your app is not available in the target repository, it will be loaded from the .github directory of the target organization's .github repository.`

To your point, the only place where I'm not checking org level configs is in "installation" webhook. So atm, users can disable product labeling at the repo level but not a the org level. This is bc in the installation (custom) handler, I can't call context.config, so I manually check configs at each repo level. 

We might want to enable product-label disabling-by-config at the org level as well, I'll add this in as a next todo. :) Thanks!